### PR TITLE
[FastReboot]: Update FR to make it working with 1.0.3

### PR DESCRIPTION
**Pull request by pavel-shirshov**

### CODE REVIEW

**Comment by pavel-shirshov** [comment]
Any comment why it was closed?

File: scripts/fast-reboot-dump.py [hunk 1]
```diff
@@ -49,24 +53,56 @@ def get_vlan_ifaces():
 
     return vlans
 
+def get_bridge_port_id_2_port_id(db):
+    bridge_port_id_2_port_id = {}
+    keys = db.keys(db.ASIC_DB, 'ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT:oid:*')
```
Comment (lguohan) [inline scripts/fast-reboot-dump.py]: how to make sure the ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT table is available be the time you run this function?

Comment (lguohan) [inline scripts/fast-reboot-dump.py]: is it possible the asic db is not completely initialized and you do not have all bridge_port id to port id mappings?

Comment (pavel-shirshov) [inline scripts/fast-reboot-dump.py]: I run this command before the switch goes into the FR. I think external tools should check a state of a switch before the initiate FR.

Comment (pavel-shirshov) [inline scripts/fast-reboot-dump.py]: It's possible, but if the device is not initialized we shouldn't reboot a device.

**Pull request merged**